Database/input-data/supplementary-source-id-info.yaml

```diff
@@ -90,13 +90,17 @@ DRES-CMIP-BB4CMIP7-2-0:
 DRES-CMIP-BB4CMIP7-2-1:
   # While we wait for ESGF scrape to come back up
   publication_status: published
+  doi: 10.25981/ESGF.input4MIPs.CMIP7/2524040
 ImperialCollege-3-0:
   # While we wait for ESGF scrape to come back up
   publication_status: published
 MRI-JRA55-do-1-6-0:
   license_id: CC BY 4.0
 PCMDI-AMIP-1-1-9:
   license_id: CC BY 4.0
+PCMDI-AMIP-1-1-10:
+  license_id: CC BY 4.0
```

Comment:
Your files don't have a `license_id` ? Probably should have caught this pre-publication, oh well one for PCMDI-AMIP-1-1-11